crates/dslab-mp/src/mc/system.rs

```diff
@@ -54,6 +55,16 @@ impl McSystem {
         }
     }
 
+    pub fn send_local_message(&mut self, node: String, proc: String, msg: Message) {
+        let event_time = Self::get_approximate_event_time(self.depth);
```

Comment:
Возможно нужно сделать вот так - но это вопрос идейный про то, считаем ли мы первые локальные события событиями модел чекинга. Мне сходу кажется, что скорее нет, но могу быть неправ

```suggestion
        self.depth += 1;
        let event_time = Self::get_approximate_event_time(self.depth);
```

Reply:
Согласен с текущим вариантом